DifferentiationInterface/docs/src/tutorial.md

```diff
@@ -6,45 +6,45 @@ CurrentModule = Main
 
 We present a typical workflow with DifferentiationInterface.jl and showcase its potential performance benefits.
 
-```@repl tuto
-using DifferentiationInterface
-import ADTypes, ForwardDiff, Enzyme
-using BenchmarkTools
+```@example tuto
+using DifferentiationInterface, ADTypes
+
+import ForwardDiff, Enzyme  # ⚠️ import the backends you want to use ⚠️
```

Comment:
maybe we could explain why we avoid using here (because there will be name conflicts with DI)